tools/python-frame-reference.py

```
@@ -51,6 +54,20 @@
 PREFIX_LEN = 7  # magic(2) + version(1) + header_len(4)
 
 
+def _load_wire_format_codec():
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `_load_wire_format_codec`

(ANN202)